spring-cloud-gcp-data-datastore/src/main/java/org/springframework/cloud/gcp/data/datastore/core/DatastoreOperations.java

```diff
@@ -57,7 +60,21 @@
 	<T> void deleteById(Object id, Class<T> entityClass);
```

Comment:
I think we need to return a `boolean` to indicate whether the delete happened.

Reply:
The delete function from the client lib is void and it says literally that deleting non-existing keys is "OK".